MVP Module/Tests/Feed UI/FeedUIIntegrationTests.swift

```diff
@@ -299,3 +327,9 @@ final class FeedUIIntegrationTests: XCTestCase {
 		return UIImage.make(withColor: .red).pngData()!
 	}
 }
+
+extension FeedViewController {
+	var errorMessage: String? {
+		return errorView?.message ?? nil
```

Comment:
If the message is `nil`, it'll already return `nil`. So the `??` is not needed

```suggestion
		return errorView?.message
```